### ISSUE TITLE
Add IMPORT INTO statement to import data by SQL using lightning physical mode

### ISSUE THREAD

**Issue by D3Hunter** [pingcap/tidb#42930]
## Feature Request

**Is your feature request related to a problem? Please describe:**

**Describe the feature you'd like:**

Add INGEST INTO statement to import data by SQL using lightning physical mode

syntax
```
IMPORT INTO tbl_name [(col_name_or_user_var [, col_name_or_user_var] ...)]
[SET col_name=expr [, col_name=expr] ...]
FROM 'file_or_url'
[FORMAT string]
[WITH option=val [, option=val]...]
```

most syntax is similar to LOAD DATA statement.

allowed `WITH` options:
- `character_set ='<string>'`
- `fields_terminated_by ='<string>'`
- `fields_enclosed_by ='<char>'`
- `fields_escaped_by ='<char>'`
- `fields_defined_null_by ='<string>'`
- `lines_starting_by ='<string>'`
- `lines_terminated_by ='<string>'`
- `skip_rows =<number>`
- `detached` to run ingest in detached mode.
- disk_quota: see lightning `disk-quota`.
- checksum_table: see lightning `checksum`.
- max_write_speed: control how fast we write
- DISABLE_TIKV_IMPORT_MODE: to avoid switching TiKV into import mode.
- thread: to control import concurrency.

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

**Comment by D3Hunter** [comment]
most code changes links to issue #40499

**Comment by D3Hunter** [comment]
this part is released in v7.2.0, so close it
